templates/include/admin.jsonnet

```diff
@@ -220,6 +229,12 @@ local common = import "common.jsonnet";
                 "secret": {
                     "secretName": "admin-internal-cert"
                 }
+            },
+            {
+                "name": "messaging-cert",
+                "secret": {
+                    "secretName": "external-certs-messaging"
```

Comment:
This should be "${MESSAGING_SECRET}" as the secret name is derived from the address space definition and passed as a template parameter.